src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -50,14 +47,6 @@ const TicketsPopover: React.FC<{ futureUpcoming: UpcomingEvent[] }> = (
   </Popover>
 );
 
-const GiftPopover = (
-  <Popover id="gift-popover">
-    <Popover.Content>
-      <GiftTicketModal />
```

Comment:
i guess you can remove GiftTicketModal as well if it's not used anywhere.

Reply:
This should've been removed... maybe a commit was left unpushed.

Reply:
Ah, OK, the component. Will do.

Reply:
removed @ 3a6c225599966a5c1851a3acdd3fed0b15bda7c8